scripts/create_dataset.py

```diff
@@ -0,0 +1,108 @@
+import os
+import argparse
+import json
+
+
+def make_new_dataset(study_name, study_description, parent_directory, output_name):
+    """
+    Gets JSON file(s) for assets, tables and relationships, combines them and then write the combined JSON to disk.
+    :param study_name: name of the dataset
```

Comment:
We should start phasing the name `study` out of our code. Use `dataset_name` instead?